src/ws2st/fs_helpers.cpp

```diff
@@ -0,0 +1,84 @@
+#include "fs_helpers.hpp"
+
+#include <iostream>
+#include <filesystem>
+#include <string>
+#include <random>
+
+namespace wash {
+
+namespace files {
+
+    namespace fs = std::filesystem;
+
+    std::string app_str;
+
+    std::string generate_random_app_str(size_t length) {
```

Comment:
Perhaps in the final version we could use this
https://en.cppreference.com/w/cpp/filesystem/temp_directory_path
But I guess we can leave it for now as it makes testing easier.

Reply:
Yeah using that path as the root for the temp directory would probably be a good idea but when debugging it's helpful to have it in the same directory my VSCode is looking at